apollo-core/src/main/java/com/ctrip/framework/foundation/internals/DefaultProviderManager.java

```diff
@@ -79,6 +80,11 @@ public String getProperty(String name, String defaultValue) {
     return defaultValue;
   }
 
+  @Override
+  public int getOrder() {
```

Comment:
as ProviderManager has a default getOrder implementation, I think we could omit this here?